servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -68,13 +68,9 @@ def update_job_labels(id, body, **kwargs):
 
     # Follow API spec
     result = response.json()
-    all_labels = get_job(id).labels
-    if not all_labels:
-        all_labels = {}
-
-    # Redundantly update all_labels with updated labels to provide consistency guarantees
-    all_labels.update(result.get('labels'))
-    return UpdateJobLabelsResponse(labels=all_labels)
+    if not result.get('labels'):
+        result.get('labels').update({})
```

Comment:
I don't think this makes sense anymore. `update()` is a method on a dict, and this if statement was attempting to catch when `labels` is null (`None` in python). Therefore this block will either never execute, or if it does, fail with method not defined. Remove or `result['labels'] = {}`

Reply:
makes sense – removed